read-fonts/src/font_data.rs

```diff
@@ -217,7 +218,9 @@ impl<'a> Cursor<'a> {
         T: FontReadWithArgs<'a> + ComputeSize,
     {
         let len = T::compute_size(args);
-        let temp = self.data.read_with_args(self.pos..self.pos + len, args);
+        let end = self.pos + Saturating(len);
```

Comment:
when we're in these methods that return `Result` my inclination would be to use checked operations and return `Errror::OutOfBounds` directly, instead of continuing with a saturated value.

Reply:
+1

Reply:
Agree.